tests/page_test.go

```diff
@@ -366,10 +363,6 @@ func TestPageReload(t *testing.T) {
 func TestPageGoBackGoForward(t *testing.T) {
 	BeforeEach(t)
 	defer AfterEach(t)
-	if isFirefox {
```

Comment:
this needs to be applied on the Go test server: https://github.com/microsoft/playwright-python/pull/282/files